src/pranitaPJun20/BankingSystem.java

```diff
@@ -0,0 +1,76 @@
+package pranitaPJun20;
+
+public class BankingSystem {
+	//Instance Variables
+	int debitCount=0;
+	int creditCount=0;
+	int printBalanceCount=0;
+	//Static variables
+	static int balance=100000;
+	static int staticDebitCount=0;
+	static int staticCreditCount=0;
+	static int staticPrintBalCount=0;
```

Comment:
Remove initialization of all instance variables. And fro initial balance you can call credit() method first.

Reply:
Done